cg/meta/orders/status.py

```diff
@@ -393,8 +400,12 @@ def store_microbial_samples(
                     ticket=ticket,
                 )
 
+                priority = new_sample.priority
```

Comment:
Looks like something is not really thought through when using this many arguments in a function call, right? Could this be split into several functions? Could we use some kind of data class to hold all this values and pass objects of that class?

Reply:
I think you are right though I don't want to do that now, this was just meant to be a small PR to allow Barry to continue his work on storing microbial analysis results for the swedac deviation. Let's save that for later.